packages/sdk/contractkit/src/wrappers/EpochManager.ts

```diff
@@ -0,0 +1,115 @@
+import { EpochManager } from '@celo/abis-12/web3/EpochManager'
+import { NULL_ADDRESS } from '@celo/base'
+import BigNumber from 'bignumber.js'
+import { proxyCall, proxySend } from './BaseWrapper'
+import { BaseWrapperForGoverning } from './BaseWrapperForGoverning'
+import { ValidatorGroupVote } from './Election'
+
+export enum EpochProcessStatus {
+  NotStarted,
+  Started,
+}
+
+export interface EpochProcessState {
+  status: EpochProcessStatus
+  perValidatorReward: BigNumber
+  totalRewardsVoter: BigNumber
+  totalRewardsCommunity: BigNumber
+  totalRewardsCarbonFund: BigNumber
+  toProcessGroups: number
+}
+
+/**
+ * Contract handling epoch management.
+ */
+export class EpochManagerWrapper extends BaseWrapperForGoverning<EpochManager> {
+  isTimeForNextEpoch = proxyCall(this.contract.methods.isTimeForNextEpoch)
+  getElected = proxyCall(this.contract.methods.getElected)
+  getEpochProcessingStatus = proxyCall(
+    this.contract.methods.epochProcessing,
+    undefined,
+    (result): EpochProcessState => {
+      return {
+        status: parseInt(result.status),
+        perValidatorReward: new BigNumber(result.perValidatorReward),
+        totalRewardsVoter: new BigNumber(result.totalRewardsVoter),
+        totalRewardsCommunity: new BigNumber(result.totalRewardsCommunity),
+        totalRewardsCarbonFund: new BigNumber(result.totalRewardsCarbonFund),
+        toProcessGroups: parseInt(result.toProcessGroups),
+      }
+    }
+  )
+
+  startNextEpochProcess = proxySend(this.connection, this.contract.methods.startNextEpochProcess)
+  finishNextEpochProcess = proxySend(this.connection, this.contract.methods.finishNextEpochProcess)
+
+  prepareFinishNextEpochProcessTx = async () => {
```

Comment:
i'd change the name here...

In reality, this is just `finishNextEpochProcess` or `completeNextEpochProcess`... it's not prepare, prepare means you're preparing and not doing anything, but we are, we end up by calling finishNextEpochProcess 

so it's confusing